cmd/zed/zed.d/zed.rc

```diff
@@ -86,6 +86,9 @@
 #
 ZED_USE_ENCLOSURE_LEDS=1
 
+##
+# Run a scrub after every resilver
+#ZED_SCRUB_AFTER_RESILVER=1
```

Comment:
Does anything actually check this variable?

Reply:
Good catch!  I'll fix that in next push